mir-module/src/main/resources/xsl/mods-seo-meta.xsl

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<xsl:stylesheet version="1.0"
+                xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+                xmlns:mods="http://www.loc.gov/mods/v3"
+                xmlns:xlink="http://www.w3.org/1999/xlink"
+                xmlns:xalan="http://xml.apache.org/xalan"
+                exclude-result-prefixes="xsl mods xlink xalan">
+
+    <xsl:include href="mods-utils.xsl"/>
+
+    <xsl:param name="CurrentLang" />
+
+    <xsl:template match="mods:mods" mode="seo-meta">
+        <xsl:apply-templates select="mods:abstract|mods:abstract/@xlink:href" mode="seo"/>
+    </xsl:template>
+
+    <xsl:template match="mods:abstract[not(@altFormat)] | mods:tableOfContents[not(@altFormat)]" mode="seo">
+        <xsl:if test="@xml:lang = $CurrentLang" >
+            <meta>
+                <xsl:attribute name="name">Description</xsl:attribute>
+                <xsl:attribute name="content">
+                    <xsl:value-of select="concat(substring(.,0,141), '...')" />
```

Comment:
A few quick points:

- I think a "horizontal ellipsis" (https://www.fileformat.info/info/unicode/char/2026/index.htm) should be used rather then three dots.
- Out of interest: Where does the target length of 140 characters come from? Is this common SEO-knowledge? Maybe add a comment.
- I think `substring(.,1,140)` would be clearer here. String indexes are 1-based in XSLT.
- The ellipsing should only be done if the processed abstract has at least that many characters.